karabo_data/geometry2.py

```diff
@@ -745,6 +754,66 @@ def from_quad_positions(cls, quad_pos, asic_gap=4, panel_gap=4):
                 ))
         return cls(modules)
 
+    @classmethod
+    def from_h5_file_and_quad_positions(cls, path, positions, unit=1e-3):
```

Comment:
Are you also planning to add a write method?

Reply:
Yes, but probably not in this pull request; I'm trying not to make it too big a change to review.